pallets/committee/src/lib.rs

```diff
@@ -395,15 +348,15 @@ decl_module! {
             proposal: T::Hash,
             index: ProposalIndex,
             approve: bool,
-        ) -> DispatchResult {
+        ) {
```

Comment:
The code-base has moved in favor of not writing it out if not necessary; I think having `Ok(())` at the end + the return type is noise as it can be assumed that all extrinsics return `DispatchResult`.